### PR TITLE
Added FeatherS2 and TinyS2 to CircuitPython supported boards

### PR DESCRIPTION
**Pull request by UnexpectedMaker**
Just adding some of my CircuitPython boards to the list of supported boards

### CODE REVIEW

File: mu/modes/circuitpython.py [hunk 1]
```diff
@@ -64,6 +64,8 @@ class CircuitPythonMode(MicroPythonMode):
         (0x04D8, 0xEAD2, None, "DynOSSAT-EDU-OBC"),
         (0x1209, 0x4DDD, None, "ODT CP Sapling M0"),
         (0x1209, 0x4DDE, None, "ODT CP Sapling M0 w/ SPI Flash"),
+        (0x239A, 0x80AC, None, "Unexpected Maker FeatherS2"),
```
Comment (carlosperate) [inline mu/modes/circuitpython.py:67]: If this is the Adafruit PID? In that case the board will be automatically detected without this addition.

File: mu/modes/circuitpython.py [hunk 2]
```diff
@@ -64,6 +64,8 @@ class CircuitPythonMode(MicroPythonMode):
...
+        (0x303A, 0x8002, None, "Unexpected Maker TinyS2"),
```
Comment (carlosperate) [inline mu/modes/circuitpython.py:68]: Was this allocated via this repo? https://github.com/espressif/usb-pids

**Review by carlosperate — COMMENTED** [review]
Thanks for the PR @UnexpectedMaker!
Added a couple of questions.

File: mu/modes/circuitpython.py (hunk 2 above)
Comment (UnexpectedMaker) [reply mu/modes/circuitpython.py:68]: Yes, TinyS2 was issued from Espressif as part of their new allocating PIDs to their customers

**Comment by ntoll** [comment]
@UnexpectedMaker thanks for the contribution. As @carlosperate pointed out, if you're just using an Adafruit VID "it should just work" :tm: In any case, happy to support new boards. Many congratulations..! :tada:

**Pull request merged**